www/src/components/Preview.tsx

```diff
@@ -10,7 +10,7 @@ export const Preview = () => {
           muted
           playsInline
           width="1200px"
-          className="border-6 rounded-lg border-[#111111] bg-[#111111] shadow-xl md:hidden"
+          className="rounded-lg border-4 border-[#111111] bg-[#111111] shadow-xl md:hidden"
```

Comment:
`border-6` doesn't exist in tailwind, so I've replaced it with `border-4`. We can also use an arbitrary value instead.